app/messengers/collection_deposit_persist_completed.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-module SubscriptionActions
+module Messengers
```

Comment:
`Messengers` doesn't mean anything to me.

Reply:
I feel like whatever these are, they're a novel concept in our work. Messengers is a blank slate and IMO so it SubscriptionActions. I'm looking for a noun that conveys an actor whose sole job is to reliably delivery messages. That's why I chose Messengers but there could be a dozen other nouns that are clearer.